app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -48,17 +48,19 @@ angular.module('miq.util').factory('chartsMixin', function() {
 
   var processHeatmapData = function(heatmapsStruct, data) {
     if (data) {
-      heatmapsStruct.data = _.sortBy(data, 'percent').map(function(d) {
-        var percent = d.percent * 100;
-        var tooltip = "Node: " + d.node + "<br> Provider: " + d.provider + "<br> Usage: " + percent + "% in use of " +
-          d.total + " total";
-
+      var heatmapsStructData = data.map(function(d) {
+        var percent = d.percent ? (d.percent * 100).toFixed(0) + '%' : 'unknown';
```

Comment:
This will break if `d.percent === 0` (because 0 is falsy). Maybe explicitly `(d.percent === null) ? ...`?